skbio/stats/tests/test_composition.py

```diff
@@ -269,6 +335,85 @@ def test_centralize(self):
                             np.array([[2, 2, 6],
                                       [4, 4, 2]]))
 
+    def test_ancom(self):
+        result = ancom(pd.DataFrame(self.table1),
+                       pd.Series(self.cats1),
+                       multiple_comparisons_correction=None)
+        W, reject = result['W'],  result['reject']
+        npt.assert_allclose(W.values,
```

Comment:
Test comparisons can be simplified (and made more robust) by creating your expected DataFrame and using `skbio.util.assert_data_frame_almost_equal`.